packages/manager/modules/pci/src/projects/project/ssh-keys/ssh-keys.controller.js

```diff
@@ -0,0 +1,39 @@
+import some from 'lodash/some';
+
+const MESSAGES_CONTAINER_NAME = 'pci.projects.project.sshKeys';
+
+export default class ProjectSshKeysController {
+  /* @ngInject */
+  constructor($translate, CucCloudMessage, OvhApiCloudProjectSshKey, projectId) {
+    this.$translate = $translate;
+    this.CucCloudMessage = CucCloudMessage;
+    this.OvhApiCloudProjectSshKey = OvhApiCloudProjectSshKey;
+    this.projectId = projectId;
+
+    this.messageHandler = CucCloudMessage.subscribe(MESSAGES_CONTAINER_NAME, {
+      onMessage: () => this.refreshMessage(),
+    });
+
+    this.$onInit();
+  }
+
+  $onInit() {
+    this.OvhApiCloudProjectSshKey
```

Comment:
According to the previous review, that promise shouldn't be returned too? 

```suggestion
    return this.OvhApiCloudProjectSshKey
```